docs/administration.landing-zones.md

```diff
@@ -16,3 +16,7 @@ Please consult the [operator documentation](./meshstack.index.md) of the respect
 
 Disabled Landing Zones can't be assigned to projects anymore. If the Landing Zone has already been assigned to a project,
 this assignment will remain, but customer admins can no longer create new projects using this Landing Zone.
+
+## Configure Landing Zone tags
+
+Platform Operators can configure a custom Landing Zone [Tag Schema](./meshstack.tag-schema.md#tag-schemas). The schema is available during the Landing Zone creation and update. For each created Landing Zone different tags can be attached.
```

Comment:
@tobiasweiss93  what do you mean by the schema is available during LZ creation and update?

Reply:
The sentence means that we show the schema during create and update. Depend on the schema we generate with ng-schema-form the form. Then we can select necessary tags.

Reply:
Aha lets change that into something users that are not aware of the JSON schema can also understand. Like "you can tag or modify tags on a landing zone during creation and edit.

Reply:
I've added your suggestion to the docs.